src/app/app-routing.module.ts

```diff
@@ -3,12 +3,16 @@ import { Routes, RouterModule } from '@angular/router';
 import { MainPageComponent } from './main-page/main-page.component';
 import { PageNotFoundComponent } from './page-not-found/page-not-found.component';
 import { FormPageComponent } from './form-page/form-page.component';
+import { SignInComponent } from './sign-in/sign-in.component';
+import { SignUpComponent } from './sign-up/sign-up.component';
 
 
 const routes: Routes = [
   { path: '', component: MainPageComponent },
   { path: '', redirectTo: '', pathMatch: 'full' },
-  {path:'form', component: FormPageComponent},
+  { path: 'form', component: FormPageComponent },
+  { path: 'signIn', component: SignInComponent },
+  { path: 'signUp', component: SignUpComponent },
   { path: '**', component: PageNotFoundComponent },  // Wildcard route for a 404 page
 ];
 
```

Comment:
Just adding routes for signIn and signUp, also reformatted the form route to be in line with the other routes.